test/dynamo/test_activation_checkpointing.py

```diff
@@ -32,12 +40,12 @@ def match_rng_op(node, op):
                 return node.args[1] == op
         return False
 
-    assert ((freq or freq_ge) and op) or ((freqs or freqs_ge) and ops)
-    if op:
+    # assert ((freq or freq_ge) and op) or ((freqs or freqs_ge) and ops)
```

Comment:
curious: why do we need to remove it?

Reply:
I wanted to have freq=0 to check that an op is absent in the graph. assert was failing there.